src/aurora/actionscript/object.cpp

```diff
@@ -65,7 +65,11 @@ Variable Object::getMember(const Variable &id) {
 		throw Common::Exception("Object::getMember id is not a string");
 
 	const Common::UString idString = id.asString();
-	if (_members.find(idString) != _members.end())
+
```

Comment:
Likewise, `getMember()` could/should be const, no?

Reply:
getMember might create new object, so i think it should stoy non const.